### ISSUE TITLE
Functional grid_sample: example of padding_mode="reflection" description error

### ISSUE THREAD

**Issue by hwangdeyu** [pytorch/pytorch#63559]
## Issue description
[torch.nn.functional.grid_sample](https://pytorch.org/docs/master/generated/torch.nn.functional.grid_sample.html#torch.nn.functional.grid_sample
)
Pick from the function description:
>padding_mode="reflection": use values at locations reflected by the border for out-of-bound grid locations. For location far away from the border, it will keep being reflected until becoming in bound, e.g., (normalized) pixel location x = -3.5 reflects by border -1 and becomes x' = 1.5, then reflects by border 1 and becomes **x'' = -0.5**.

## Expected description
x = **-0.5** should be x = **0.5**

**Comment by mruberry** [comment]
Thanks for reporting this issue, @hwangdeyu
